wayang-applications/data/case-study/README.md

```diff
@@ -0,0 +1,3 @@
+# Blossom Sky Setup on the "central node"
```

Comment:
Please change into Wayang, Blossom Sy is the old name of Scalytics.

Reply:
Removed - was an old artifact.